### DISCUSSION TITLE
Support right-click context menu

### DISCUSSION THREAD

**Discussion by Erudition** [webstudio-is/webstudio-community#88] [Ideas]
From other design tools I'm used to be able to right-click things to get options, but in Webstudio I currently just get the browser context menu (and sometimes it takes a second to realize that none of those options apply)

**Comment by kof** [comment]
Definitely!

**Comment by BeanIamonte** [comment]
Hi! I agree that right-clicking would be a great update

I find it super useful for resetting a CSS value. (I often have to wait 2 seconds when I want to remove a variable because I don't like the design).

 I would find it great if a simple right click on the CSS property would reset it.
